ui/src/app/app.module.ts

```diff
@@ -117,12 +119,13 @@ export function getConfiguration(signInService: SignInService): Configuration {
       useClass: ErrorReporterService,
     },
     SignInService,
+    StatusCheckService,
     GoogleAnalyticsEventsService,
     {
       provide: Http,
       useClass: InterceptedHttp,
       deps: [XHRBackend, RequestOptions, ErrorHandlingService]
-    },
```

Comment:
No need to fix, but when the language syntax allows for trailing commas, I tend to add them so single-line additions don't result in multi-line code changes.